kms/kms.rb

```diff
@@ -174,18 +174,90 @@
       Cloudkms::CryptoKey.new(purpose: "ENCRYPT_DECRYPT")
   )
 
-  puts "Created version #{crypto_key_version.name} for key " +
-       "#{crypto_key} in key ring #{key_ring_id}"
+  version = crypto_key_version.name.split("/").last
```

Comment:
crypto_key_version.name contains a the full resource path to the crypto key version. I'm just splitting the path by "/" and getting the last element, the version number.